src/Select.js

```diff
@@ -653,6 +657,16 @@ export default class Select extends Component<Props, State> {
   // Getters
   // ==============================
 
+  getTheme() {
+    if (this.props.theme) {
+      if (typeof this.props.theme === 'function') {
+        return this.props.theme(defaultTheme);
+      }
+      return this.props.theme;
```

Comment:
Returning instead `Object.assign({}, defaultTheme, this.props.theme)` here would guarantee that all of the default theme values are included, rather than trusting and requiring that a custom theme takes care of them. That would simplify the requirements on a custom theme to just define the changed variables, and would help with backwards-compatibility when the default theme will eventually change.

Reply:
Do you mean also removing the function form of the prop? I'm okay with that too, but it'd mean people have to import the default theme.

Reply:
No, I meant just replacing line 665 with the `Object.assign` statement. The function form makes sense as a route for "advanced" theming, and with it the default theme itself wouldn't even need to be exported from the package.

Reply:
Gotcha. Fixed :)